examples/complete/main.tf

```diff
@@ -31,4 +31,5 @@ module "set_quota" {
   source            = "../../modules/quotas"
```

Comment:
additional cleaning: 
we can remove unnecessary double slashes from `source = "../..//modules/plan"`